### PR TITLE
[LGS-0001] initialize login service

### PR DESCRIPTION
**Pull request by mwidyr**
initialize login service module, only have signup, login and authorize when get user profile

### CODE REVIEW

**Comment by isfanazha** [comment]
Why not use another service name instead of loginservice name? Maybe like userservice, oauthservice, or anything else? WDYT? @mwidyr

File: svc/loginservice/internal/delivery/rest/echo/init.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	e := echo.New()
+	e.Use(middleware.LoggerWithConfig(middleware.LoggerConfig{
+		Format: "time=${time_rfc3339}, method=${method}, uri=${uri}, status=${status} latency=${latency_human}\n",
+	}))
+
+	v1 := e.Group("/v1")
+
+	login := service.NewLogin(app.UseCases.LoginUC)
+	v1.Add(http.MethodGet, "/products", login.GetProducts)
+	v1.Add(http.MethodGet, "/products/:id", login.GetProduct)
```
Comment (isfanazha) [inline svc/loginservice/internal/delivery/rest/echo/init.go]: why we need those endpoints?

File: svc/loginservice/cmd/loginservice/main.go [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	if env == "" {
+		env = constant.OsEnvDefaultEnvironment
+	}
+
+	//configPath := flag.String("config", "", "specifies configuration's file path")
+	flag.Parse()
+
+	log.SetFlags(log.Ldate | log.Ltime | log.Lmicroseconds | log.Llongfile)
+
+	prodServiceApp, err := app.Newloginservice(constant.ConfigProjectFilepath)
```
Comment (isfanazha) [inline svc/loginservice/cmd/loginservice/main.go]: I think this is not a product service right? wkwkwk :smile:

File: svc/loginservice/internal/delivery/rest/echo/init.go (hunk 1 above)
Comment (mwidyr) [reply svc/loginservice/internal/delivery/rest/echo/init.go]: wkwk still dummy from product module

File: svc/loginservice/cmd/loginservice/main.go (hunk 2 above)
Comment (mwidyr) [reply svc/loginservice/cmd/loginservice/main.go]: okaay

**Comment by mwidyr** [comment]
> Why not use another service name instead of loginservice name? Maybe like userservice, oauthservice, or anything else? WDYT? @mwidyr

hmm, because i think login should be separated from user, so it only serve for login/signup. there will be another service which edit user information, admin access and etc @isfanazha

**Review by isfanazha — COMMENTED** [review]
LGTM!

**Review by isfanazha — APPROVED** [review]

**Comment by mwidyr** [comment]
i change the module to user service first, because there are something that i want to try.

**Pull request merged**